src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -229,51 +222,6 @@ public enum MicrobeSymmetry
     [JsonIgnore]
     public MicrobeCamera Camera => camera;
 
-    /// <summary>
-    ///   The selected membrane rigidity
-    /// </summary>
-    [JsonIgnore]
-    public float Rigidity
-    {
-        get => rigidity;
-        set
-        {
-            rigidity = value;
-
-            if (previewMicrobe?.Species != null)
-            {
-                previewMicrobe.Species.MembraneRigidity = value;
-                previewMicrobe.ApplyMembraneWigglyness();
-            }
-        }
-    }
-
-    /// <summary>
-    ///   Selected membrane type for the species
-    /// </summary>
-    [JsonProperty]
-    public MembraneType Membrane { get; private set; }
-
-    /// <summary>
-    ///   Current selected colour for the species.
-    /// </summary>
-    [JsonIgnore]
-    public Color Colour
-    {
-        get => colour;
-        set
-        {
-            colour = value;
-
-            if (previewMicrobe?.Species != null)
-            {
-                previewMicrobe.Species.Colour = value;
-                previewMicrobe.Membrane.Tint = value;
-                previewMicrobe.ApplyPreviewOrganelleColours();
```

Comment:
Removing these broke the live preview from showing the colour and rigidity slider changes.